controllers/object/SplatTask.js

```diff
@@ -62,13 +65,23 @@ class SplatTask {
                 method: 'POST',
                 headers: {
                     ...form.getHeaders(),
+                    Authorization: `Bearer ${this.userJWT}`,
                 },
                 body: form,
             });
 
-            const gaussianSplatRequestId = await res.text();
-            this.object.gaussianSplatRequestId = gaussianSplatRequestId;
-            this.gaussianSplatRequestId = gaussianSplatRequestId;
+            let responseText = await res.text();
+            try {
+                // response is a string if successful, a JSON {error: 'reason'} if not
```

Comment:
The API being shaped like this is less than ideal. I'm guessing it's out of scope to change it now but expecting a parse exception isn't good